### PR TITLE
Add side nav for admin cluster

### PR DESCRIPTION
**Pull request by Mubangizi**
#### What does this PR Do?
Adds side nav for the cluster admin.
#### Description
Resource side nav added it to app page
configured page look for cluster page
#### Screen Shots
Screenshot from 2020-02-11 16-22-11

### CODE REVIEW

File: src/components/SideNav/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  },
+];
+
+function SideNav() {
+  return (
+    <div className='sidenav'>
+      <div className="list">
+        {items.map((item) =>
+          <div key={item.name}>
+            <div key={items.indexOf(item)} className="list-item">{item.name}</div>
```
Comment (neelxie) [inline src/components/SideNav/index.js]: @Mubangizi good stuff kindly use Pascalcase for class names here.

File: src/components/SideNav/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <div key={items.indexOf(item)} className="">
+              {item.items.map((subitem) =>
+                <div key={items.indexOf(subitem)} className="sub-list-Item">{subitem.name}</div>
```
Comment (neelxie) [inline src/components/SideNav/index.js]: Even here and other relevant places.

**Comment by Mubangizi** [comment]
Yeah Thanks @StephenAraka  and @neelxie  i just understood what Pascal case was like 20 minutes ago am on it though

**Comment by MutegekiHenry** [comment]
@Mubangizi resolve the conflicts concerning the linter `.eslintrc.json`

**Pull request merged**